pkg/jx/cmd/install.go

```diff
@@ -1109,13 +1114,13 @@ func (options *InstallOptions) Run() error {
 		}
 
 		gitIgnore := filepath.Join(gitOpsDir, ".gitignore")
-		err = ioutil.WriteFile(gitIgnore, []byte(devGitOpsGitIgnore), util.DefaultWritePermissions)
+		err = secretStore.Write(gitIgnore, []byte(devGitOpsGitIgnore))
```

Comment:
This could probably do with:
a) Renaming from `secretStore` (as it's just a more generic store that stores to the filesystem but will be extended to (optionally) store in vault); and
b) merging with your `io` package (that I didn't know existed at the time).

the current `secretStore` is more a generic abstraction of somewhere to store config (badly named atm)